packages/test/test-end-to-end-tests/src/test/gc/gcTombstoneDataStores.spec.ts

```diff
@@ -58,8 +58,9 @@ describeNoCompat("GC data store tombstone tests", (getTestObjectProvider) => {
         if (provider.driver.type !== "local") {
```

Comment:
There should be tests that validate usage of this. I believe your [other PR](https://github.com/microsoft/FluidFramework/pull/13606) will have it but mentioning here.

Reply:
Yeah let me add tests here for this change itself

Reply:
@agarwal-navin done